### PR TITLE
12546-FSS-UI Refresh Batch Attributes Stored In Local Storage On Token Refresh Changes

### PR DESCRIPTION
**Pull request by gaurav15233**
Hi All, Please review and merge changes related to the refresh batch attributes stored in local storage on token refresh changes for FSS UI in main branch.

### CODE REVIEW

**Review by jimjamjahaa — CHANGES_REQUESTED** [review]
just 2 small changes required

**Review by bhalcha13813 — APPROVED** [review]
LGTM

**Comment by gaurav15233** [comment]
Hi Richard, Thanks for reviewing PR. I have done the changes as per comments. Can you please review this.

**Review by jimjamjahaa — APPROVED** [review]
lgtm

File: src/app/features/fss-search/fss-search.component.ts [hunk 1]
```diff
@@ -297,10 +299,23 @@ export class FssSearchComponent implements OnInit {
         this.displayLoader = false;
         this.analyticsService.login();
       })
+      this.fileShareApiService.getBatchAttributes().subscribe((batchAttributeResult) => {
```
Comment (crdandoukho) [inline src/app/features/fss-search/fss-search.component.ts:302]: Can the code in the new subscription for the `getBatchAttributes` event be performed in the existing event handler  on line 82?

**Review by crdandoukho — COMMENTED** [review]
Just one question re the duplicated event subscription

File: src/app/features/fss-search/fss-search.component.ts (hunk 1 above)
Comment (crdandoukho) [reply src/app/features/fss-search/fss-search.component.ts:302]: Forget that. After debugging I see I have misunderstood how `subscribe()` is working and the callback on line 82 is not called after the log-in pop up

**Comment by crdandoukho** [comment]
@gaurav15233 could you please resolve the merge conflicts?

**Comment by gaurav15233** [comment]
Hi Chris, Thanks for reviewing this PR. I have resolved the merge conflicts. Can you please review again.

**Pull request merged**